scss/_forms.scss

```diff
@@ -4,6 +4,7 @@
 
 .form-control {
   display: block;
+  font-weight: $input-font-weight;
```

Comment:
Properties should be ordered display, width, padding, font-size, font-weight, line-height, color, background-color, background-image, border